stellar-multitenant/pkg/tenant/manager.go

```diff
@@ -117,10 +169,15 @@ func (m *Manager) GetTenantByIDOrName(ctx context.Context, arg string) (*Tenant,
 
 // GetDefault returns the tenant where is_default is true. Returns an error if more than one tenant is set as default.
 func (m *Manager) GetDefault(ctx context.Context) (*Tenant, error) {
-	var tnts []Tenant
-	query := fmt.Sprintf(selectQuery, "WHERE is_default = true")
+	queryParams := &QueryParams{
+		Filters: map[FilterKey]interface{}{
+			FilterKeyIsDefault: true,
+		},
+	}
```

Comment:
👋 🤔  What happens if you try to deactivate a default tenant?

Reply:
addressed in [993ea99](https://github.com/stellar/stellar-disbursement-platform-backend/pull/253/commits/993ea99d2a28bdae1e02e8c03d9b44abc85cb384)